src/docfx/lib/UrlUtility.cs

```diff
@@ -159,8 +162,14 @@ public static string GetRelativeUrl(string relativeToUrl, string url)
                 result.Append(path, segmentIndex, path.Length - segmentIndex);
             }
 
-            result.Append(query);
-            result.Append(fragment);
+            if (!string.IsNullOrEmpty(query))
```

Comment:
what's the difference? `query` and `fragment` should never be null

Reply:
There was 1 commit `query` and `fragment` could be null, they cannot now.